### DISCUSSION TITLE
flask crashed

### DISCUSSION THREAD

**Discussion by themeswordpress** [pallets/flask#4796] [Q&A]
Dears,

Hope all is well. 

I have a project, you can say a big project has more than 168 API routes (/x /y /yyy ..etc) 

each of these API routes has a specific function in my project I am facing a big problem that the flask crashed or the flask slower after sending about 1500 - 2000 req/second.

and I notched that the system resource not consumed (I,e my RAM 8 GB the total used is 4 GB and CPU utilization is 40 to 65%) 

so what the issue here? 

and do you have real-world examples company is using your API for a big project?

and what is your recommendations  for the production apps

**Comment by robertlagrant** [comment]
@themeswordpress some questions, not answers :) 

- what was the reported error when it crashed?
- if you test a single endpoint app with the same environment, and the same requests/second, do you still get the issue?
- what sort of work are your endpoints doing?
